pokeapp/src/components/GamePage.js

```diff
@@ -26,13 +34,142 @@ const GamePage = ({makeUpperCase, setIsPokeList}) => {
         });
       };
 
+// ------------------------------------------------------------------------------------------------SLOT MACHINE LOGIC START
+
+      const rollerStyle = {
+        backgroundImage: `url(${pikachu})`,
+      }
+
+      const rollerStyle2 = {
+        backgroundImage: `url(${pokeball})`,
+      }
+
+      const roller1 = () => {
+
+        let reelStrip = [];
+        if (reel1) {
+          for (let current = 1; current <= NUMBER_OF_REEL_FACES; current++ ) {
+            reelStrip.push(
+              <>
+              <div style={rollerStyle} className="roller-img roller-ani"/>
+              <div style={rollerStyle2} className="roller-img roller-ani"/>
+              </>
```

Comment:
We might need to key this block.  Are you getting console warnings about it?

Reply:
Definitely console warnings about a key prop. This is getting pushed into the reel strip from a loop so would it be ok to use the "current" value in the for loop (line 51 for example)?

Reply:
Maybe on line 53, put `<React.Fragment>` with a `key` inside to make this work without warnings????

Reply:
@BlackCubes That was exactly the fix! Thank you.